src/exchanges/bitfinex-client.js

```diff
@@ -207,9 +234,79 @@ class BitfinexClient extends BasicClient {
   }
 
   _onTradeMessage(msg, market) {
-    // example msg: [ 359491, 'tu', [ 560287312, 1609712228656, 0.005, 33432 ], 6 ]
-    let [id, unix, amount, price] = msg[2];
+    const timestampMs = msg[3];
+    if (msg[1] === "hb") {
```

Comment:
Suggest extracting to `_onTradeHeartbeat` and adding method comment with message JSON